client/deposits/utils/index.js

```diff
@@ -129,9 +129,10 @@ export const getDepositScheduleDescriptor = ( {
 		);
 	}
 
+	const learnMoreHref =
```

Comment:
Considering this is a constant for the whole method, should we move it next to `isCustomDepositSchedulesEnabled` and `hasCompletedWaitingPeriod` in the beginning of the code block?